app/src/main/java/com/android/voyageur/model/place/GooglePlacesRepository.kt

```diff
@@ -108,28 +107,13 @@ class GooglePlacesRepository(private val placesClient: PlacesClient) : PlacesRep
 
     // Define which fields to retrieve for each place
     placeIds.forEach { placeId ->
-      val request = FetchPlaceRequest.builder(placeId, placeFields).build()
+      val request = FetchPlaceRequest.builder(placeId, freePlaceFields).build()
       placesClient
           .fetchPlace(request)
           .addOnSuccessListener { response ->
             val place = response.place
-            val bitmaps = mutableListOf<ImageBitmap>()
-
-            val photosMetadata = ((place.photoMetadatas) ?: emptyList<PhotoMetadata>()).take(5)
-            for (photoMetadata in photosMetadata) {
-              val photoRequest = FetchPhotoRequest.builder(photoMetadata).build()
-              placesClient
-                  .fetchPhoto(photoRequest)
-                  .addOnSuccessListener { fetchPhotoResponse ->
-                    val bitmap = fetchPhotoResponse.bitmap.asImageBitmap()
-                    bitmaps.add(bitmap)
-                  }
-                  .addOnFailureListener { exception ->
-                    Log.e("PlacesRepository", "Failed to fetch photo for $placeId", exception)
-                  }
-            }
+            val bitmaps = emptyList<ImageBitmap>()
             places.add(CustomPlace(place, bitmaps))
```

Comment:
maybe now we don't need a custom place out of the box / we don't require the images(bitmaps) at this steps as they will get filled later.

Reply:
I think we will need the data class when we support adding places to a trip, in order to also display photos there. This class would allow to store the images and not fetch them each time.

Reply:
Hmm, good point. We should talk about this in the next standup, maybe there is no use for the images, but just in case keep it like this and we'll see in the future if we need to remove them or not. Thanks for the quick reply and great work!

Reply:
However, maybe it would be beneficial to get the line coverage above 80%. I think the view model tests could be updated a bit.

Reply:
Yes forgot to update the PlacesViewModel tests, tx for catching that :)